migtests/scripts/live-migration-fallb-run-test.sh

```diff
@@ -21,6 +21,19 @@ export TEST_DIR="${TESTS_DIR}/${TEST_NAME}"
 export QUEUE_SEGMENT_MAX_BYTES=400
 export PYTHONPATH="${REPO_ROOT}/migtests/lib"
 
+run_via_config_file=false
+
+# Parse optional second argument
+if [ $# -eq 2 ]; then
+    if [ "$2" = "--run-via-config-file" ]; then
+        run_via_config_file=true
+    else
+        echo "Unknown option: $2"
+        echo "Usage: $0 TEST_NAME [--run-via-config-file]"
+        exit 1
+    fi
+fi
```

Comment:
Tech Debt: It will better to have same set of arguments in all the run script unless something very customised for a script.
Also in that case, the set of mandatory or initial args can be same, and extra ones can be later in order.